src/overrides/venia-ui/components/ProductFullDetail/productFullDetail.js

```diff
@@ -1,28 +1,38 @@
-import React, { Fragment, Suspense } from 'react';
-import { FormattedMessage, useIntl } from 'react-intl';
-import { arrayOf, bool, number, shape, string } from 'prop-types';
-import { Form } from 'informed';
-import { Info } from 'react-feather';
+import React, {Fragment, Suspense} from 'react';
+import {FormattedMessage, useIntl} from 'react-intl';
+import {arrayOf, bool, number, shape, string} from 'prop-types';
+import {Form} from 'informed';
+import {Info, ShoppingBag} from 'react-feather';
 
 import Price from '@magento/venia-ui/lib/components/Price';
-import { useProductFullDetail } from '@magento/peregrine/lib/talons/ProductFullDetail/useProductFullDetail';
-import { isProductConfigurable } from '@magento/peregrine/lib/util/isProductConfigurable';
+import {useProductFullDetail} from '@magento/peregrine/lib/talons/ProductFullDetail/useProductFullDetail';
+import {isProductConfigurable} from '@magento/peregrine/lib/util/isProductConfigurable';
 
-import { useStyle } from '/var/www/learning3/pwa-learning/node_modules/@magento/venia-ui/lib/classify.js';
-import Breadcrumbs from '/var/www/learning3/pwa-learning/node_modules/@magento/venia-ui/lib/components/Breadcrumbs/index.js';
-import Button from '/var/www/learning3/pwa-learning/node_modules/@magento/venia-ui/lib/components/Button/index.js';
-import Carousel from '/var/www/learning3/pwa-learning/node_modules/@magento/venia-ui/lib/components/ProductImageCarousel/index.js';
-import FormError from '/var/www/learning3/pwa-learning/node_modules/@magento/venia-ui/lib/components/FormError/index.js';
-import { QuantityFields } from '/var/www/learning3/pwa-learning/node_modules/@magento/venia-ui/lib/components/CartPage/ProductListing/quantity.js';
-import RichContent from '/var/www/learning3/pwa-learning/node_modules/@magento/venia-ui/lib/components/RichContent/richContent.js';
-import { ProductOptionsShimmer } from '/var/www/learning3/pwa-learning/node_modules/@magento/venia-ui/lib/components/ProductOptions/index.js';
+import {useStyle} from '@magento/venia-ui/lib/classify.js';
+import Breadcrumbs
+    from '@magento/venia-ui/lib/components/Breadcrumbs/index.js';
+import Button from '@magento/venia-ui/lib/components/Button/index.js';
+import Carousel
+    from '@magento/venia-ui/lib/components/ProductImageCarousel/index.js';
+import FormError
+    from '@magento/venia-ui/lib/components/FormError/index.js';
+import {
+    QuantityFields
+} from '@magento/venia-ui/lib/components/CartPage/ProductListing/quantity';
+import RichContent
+    from '@magento/venia-ui/lib/components/RichContent/richContent.js';
+import {
+    ProductOptionsShimmer
+} from '@magento/venia-ui/lib/components/ProductOptions/index.js';
 import CustomAttributes from './CustomAttributes';
 import defaultClasses from './productFullDetail.module.css';
 import CmsBlock from '@magento/venia-ui/lib/components/CmsBlock';
-import {TabGroup} from '../../../../components/Tabs/tabs';
+import Tabs from "../../../../components/NewTabs/tab";
```

Comment:
`import Tabs from "../../../../components/NewTabs";`

Reply:
Please add index.js file for your NewTabs component where you would have smth like this:

```
export { default as Tabs } from './tab';
export { default as TabPane } from './tab-pane';
```